cost/aws/instance_cloudwatch_utilization/README.md

```diff
@@ -1,83 +1,85 @@
-```json
-	"metrics": {
-		"append_dimensions": {
-			"AutoScalingGroupName": "${aws:AutoScalingGroupName}",
-			"ImageId": "${aws:ImageId}",
-			"InstanceId": "${aws:InstanceId}",
-			"InstanceType": "${aws:InstanceType}"
-    }
-  }
-```
-#### Input Parameters
+If you get an **N/A** in a field you will need to install the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html) on the instance to get those metrics.
+
+## Policy Actions
+
+- Sends an email notification
+- Downsizes instances after approval
+
+## Prerequisites
+
+This policy requires the AWS IAM User Credential. When applying the policy select the appropriate credentials
+from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
+
+The credential must contain the value *AWS* in the Provider field.
+Refer to our documentation for more details on the [Credential Service](https://docs.rightscale.com/credentials/)
+
+## Input Parameters
 
 This policy has the following input parameters required when launching the policy.
 
-- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
-- *Average used memory percentage* - Utilization below this percentage will raise an incident to tag the instance. Providing -1 will turn off this metric for consideration.
+- *Email addresses of the recipients you wish to notify*
+- A list of email addresses to notify- *Average used memory percentage*
+- Utilization below this percentage will raise an incident to tag the instance. Providing -1 will turn off this metric for consideration.
 - *Average used CPU percentage* - Utilization below this percentage will raise an incident to tag the instance. Providing -1 will turn off this metric for consideration.
 - *Exclusion Tag Key* - An AWS-native instance tag to ignore instances that you don't want to consider for downsizing. Only supply the tag key
-- *Action Tag Key:Value* - The tag key:value pair to set on an instance that is underutilized.
 
-### AWS Required Permissions
+## Required Permissions
 
-This policy requires permissions to list Metrics and Get Metric Statistics from the AWS Cloudwatch API.
-The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. The IAM user credentials contained in those credentials will require the following permissions:
+### AWS Required Permissions
 
-```javascript
-{
+This policy requires permissions to list Metrics and Get Metric Statistics from the AWS Cloudwatch API.The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. The IAM user credentials contained in those credentials will require the following permissions:
```

Comment:
Remove the 2nd sentence from the paragraph.  Replace IAM user with AWS.

Reply:
Updated README as mentioned in the comment